guessingGame.py

```diff
@@ -45,7 +45,9 @@ def guessing_game_function():
     guessing_game_function()
 
     answer = input("Do you wish to play again? YES or NO: ")
-    if answer == "YES" or answer == "Yes" or answer == "yes":
+    if answer == "YES":

```

Comment:
No. I meant you should convert the answer variable to lower or upper case and compare it to the string i.e YES/yes in lower/upper case